drivers/usb/usbohci_new/usbohci.h

```diff
@@ -64,7 +65,11 @@ typedef struct _OHCI_HCD_TD {
   ULONG Pad[1];
 } OHCI_HCD_TD, *POHCI_HCD_TD;
 
+#ifdef _WIN64
+C_ASSERT(sizeof(OHCI_HCD_TD) == 96);
+#else
 C_ASSERT(sizeof(OHCI_HCD_TD) == 64);
+#endif
```

Comment:
This structure is not hardware-dependent?

Reply:
Does `LIST_ENTRY DoneLink;` look hardware dependent?

Reply:
Ah ok nope.